corc-core/src/main/java/com/hotels/corc/mapred/CorcInputFormat.java

```diff
@@ -303,4 +314,11 @@ static private int getOrcAtomicRowColumnId() {
 
   }
 
+  public static String toKryo(SearchArgument sarg) {
```

Comment:
Add a comment like in the class header comments saying exactly where this code came from, it's more likely to be read by future developers here.